### ISSUE TITLE
Feature request: Alias for subcommands

### ISSUE THREAD

**Issue by atticus-sullivan** [alexflint/go-arg#230]
It would be nice to be able to define aliases for subcommands. In my case I have subcommands for `remove` and `list` and it would be nice to be able to shorten this to `rm` and `ls` to save time typing while still accept the longer forms for better readability.

Maybe something like (added `r` and `l` as aliases on top to show how to do this without the limitation of only one single alias)
```go
var args struct {
	Remove *RemoveCmd `arg:"subcommand:list,alias:rm:r"`
	List   *ListCmd   `arg:"subcommand:list,alias:ls,l"`
}
```

Of course one could define a new subcommand of the same type with the aliased name, but that wouldn't look nice in the help string, blows up the code (additional field + additional check when checking what subcommand was selected) and doesn't quite capture the intended meaning.

**Comment by alexflint** [comment]
Yes that would be nice! In terms of details, any reason not to make the `subcommand:` part into a list, where the first one shows up as the default, and there is then no need for a separate `alias:` part:

```go
var args struct {
	Remove *RemoveCmd `arg:"subcommand:remove,rm,r"`
	List   *ListCmd   `arg:"subcommand:list,ls,l"`
}
```

**Comment by atticus-sullivan** [comment]
Thanks for the quick fix :+1:

**Comment by alexflint** [comment]
You got it! I haven't got a release for this feature yet but you should be able to get it with the git commit based version tag thing that go has. Let me know if you're not familiar with this thing and I can find a pointer.

**Comment by atticus-sullivan** [comment]
I used `go get github.com/alexflint/go-arg@bf629a16cbe505e81b6347813a918770a3b727df` and it works like a charm. Thanks again :+1:
